plays/dc1/swarm-bootstrap.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Nah koyok ngene, awamu kan nulis maneh `Ensure python is present`

mending langsung dadino role

terus panggil sesuai urutan